docs/source/installation.rst

```diff
@@ -179,7 +179,6 @@ In the ``anaconda prompt`` terminal:
 
 6. Build an editable version of sktime :code:`pip install -e .[dev]`
 In order to install all soft dependencies, use :code:`pip install -e .[dev]`
-If you are on Mac, use :code:`pip install -e ".[dev]"`
```

Comment:
I think it was a very good thought to add the point earlier, becuase a lot of users may face that error. I'll suggest to add it back without specifying Mac, or may be by adding a note something on the lines of:

> In case you get an error saying `no matches found: sktime[dev]`, use `pip install -e ".[dev]"`

Reply:
it's already in the troubleshooting section, we could have it twice though (at the point where it occurs).
Or just point to the troubleshooting section

use -> try?

Reply:
sure!